ERROR IN PRODUCTION MAGSTOCK

```
Traceback Detected
Traceback Detected

Location: magstock7.uber.magfest.org    Time: 2017-04-02T17:19:01.191Z
2017-04-02 13:18:58,572 [ERROR] uber.decorators: Exception raised, but we're going to ignore it and continue.
   Traceback (most recent call last):
     File "/usr/local/uber/plugins/uber/uber/decorators.py", line 253, in with_session
       retval = func(*args, session=session, **kwargs)
     File "/usr/local/uber/plugins/uber/uber/decorators.py", line 381, in with_restrictions
       return func(*args, **kwargs)
     File "/usr/local/uber/plugins/uber/uber/decorators.py", line 343, in with_rendering
       result = func(*args, **kwargs)
     File "/usr/local/uber/plugins/uber/uber/decorators.py", line 85, in protected
       return func(*args, **kwargs)
     File "/usr/local/uber/plugins/uber/uber/site_sections/groups.py", line 131, in delete
       raise HTTPRedirect('index?message={}', 'Group deleted')
   uber.utils.HTTPRedirect: (['https://magstock7.uber.magfest.org/uber/groups/index?message=Group%20deleted'], 302)
   
   During handling of the above exception, another exception occurred:
   
   Traceback (most recent call last):
     File "/usr/local/uber/plugins/uber/uber/decorators.py", line 13, in swallow_exception
       return func(*args, **kwargs)
     File "/usr/local/uber/plugins/uber/uber/models.py", line 2164, in _track_changes
       Tracking.track(action, instance)
     File "/usr/local/uber/plugins/uber/uber/models.py", line 2098, in track
       _insert(instance.session)
     File "/usr/local/uber/plugins/uber/uber/models.py", line 2095, in _insert
       snapshot=json.dumps(instance.to_dict(), cls=serializer)
     File "/usr/lib/python3.4/json/__init__.py", line 237, in dumps
       **kw).encode(obj)
     File "/usr/lib/python3.4/json/encoder.py", line 192, in encode
       chunks = self.iterencode(o, _one_shot=True)
     File "/usr/lib/python3.4/json/encoder.py", line 250, in iterencode
       return _iterencode(o, 0)
     File "/usr/local/uber/sideboard/lib/_utils.py", line 45, in default
       raise json.JSONEncoder.default(self, o)
     File "/usr/lib/python3.4/json/encoder.py", line 173, in default
       raise TypeError(repr(o) + " is not JSON serializable")
   TypeError: Decimal('0') is not JSON serializable
```

--- a/uber/__init__.py
+++ b/uber/__init__.py
@@ -1,2 +1,10 @@
+from decimal import Decimal
+
 from uber._version import __version__
 from uber.common import *
+
+from sideboard import lib
+
+
+# NOTE: this will decrease the precision of some serialized decimal.Decimals
+lib.serializer.register(Decimal, lambda n: float(n))

--- a/uber/tests/test_sideboard_serializer.py
+++ b/uber/tests/test_sideboard_serializer.py
@@ -1,0 +1,16 @@
+from decimal import Decimal
+
+import pytest
+
+from uber.common import *
+
+
+@pytest.mark.parametrize('test_input,expected', [
+    (Decimal(), '0.0'),
+    (Decimal(0), '0.0'),
+    (Decimal(0.1), '0.1'),
+    ([Decimal(0), Decimal(0.1)], '[0.0, 0.1]'),
+    ({'d': Decimal(0.1)}, '{"d": 0.1}')
+])
+def test_decimal(test_input, expected):
+    assert expected == json.dumps(test_input, cls=serializer)

